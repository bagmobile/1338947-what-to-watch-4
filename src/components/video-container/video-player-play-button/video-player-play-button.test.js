import React from "react";
import renderer from "react-test-renderer";
import VideoPlayerPlayButton from "./video-player-play-button";

describe(`VideoPlayerPlayButton component`, () => {

  it(`Render`, () => {
    const tree = renderer.create(
        <VideoPlayerPlayButton
          onClick={() => {}}
        />);
    expect(tree).toMatchSnapshot();
  });

});
