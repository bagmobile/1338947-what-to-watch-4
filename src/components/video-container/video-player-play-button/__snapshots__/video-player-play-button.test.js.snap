// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerPlayButton component Render 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
